.eslintrc

```diff
@@ -18,6 +18,6 @@
     "ReactDOM": "readonly",
     "PropTypes": "readonly",
     "ReactRouterDOM": "readonly",
-    "mailtouiApp": "readonly",
+    "mailtouiApp": "readonly"
```

Comment:
nit: can we keep the ,?

It makes diffs cleaner. Unless the linter's automatically removing it.

Reply:
sure thing, I didn't see this in the other PR haha.
my bad